projects/sandbox/data_generation/data_generation/utils.py

```diff
@@ -93,7 +93,7 @@ def download_data(
     return data.resample(sample_rate)
 
 
-def inject_into_background(
+def inject_into_random_background(
```

Comment:
wondering why the name change here, is this doing anything functionally different now?